frontends/election-manager/src/screens/print_test_deck_screen.tsx

```diff
@@ -215,14 +278,21 @@ export function PrintTestDeckScreen(): JSX.Element {
             ))}
         </ButtonList>
       </NavigationScreen>
-      {precinctIndex !== undefined && (
-        <TestDeckBallotsMemoized
-          election={election}
-          electionHash={electionHash}
-          precinctId={precinctIds[precinctIndex]}
-          onAllRendered={onAllRendered}
-        />
-      )}
+      {printIndex !== undefined &&
+        (printIndex.sectionIndex === 0 ? (
+          <OneSidedMaterials
+            election={election}
+            precinctId={precinctIds[printIndex.precinctIndex]}
+            onAllRendered={onAllOneSidedMaterialsRendered}
+          />
+        ) : (
+          <TwoSidedMaterialsMemoized
```

Comment:
Given this discussion in Slack (https://votingworks.slack.com/archives/CEL6D3GAD/p1652996981036299), we may actually want to define these components not in terms of one-sided/two-sided but in terms of contents, e.g. `<HmpbMaterials>`, `<BmdMaterials>`, `<TallyReports>`